x/margin/keeper/params.go

```diff
@@ -7,7 +7,9 @@ import (
 
 // GetParams get all parameters as types.Params
 func (k Keeper) GetParams(ctx sdk.Context) types.Params {
-	return types.NewParams()
+	var params types.Params
+	k.paramstore.GetParamSet(ctx, &params)
```

Comment:
This paramstore is used until SDK 47, and from SDK 47, just normal store is used as other keys I think

Reply:
@kenta-elys any updates on this change?

Reply:
@cosmic-vagabond Done.